account_invoice_payment_term_date_due/views/account_move.xml

```diff
@@ -5,23 +5,23 @@
     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 -->
 <odoo>
-
     <record id="view_move_form" model="ir.ui.view">
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
         <field name="arch" type="xml">
-            <field name="invoice_payment_term_id" position="after">
-                <span
-                    name="invoice_payment_term_date_due"
-                    class="text-muted oe_read_only"
+            <xpath expr="//field[@name='invoice_payment_term_id']" position="after">
+                <i
+                    class="fa fa-arrow-right mx-3 text-muted"
```

Comment:
```
2023-05-29 15:48:21,047 254 WARNING odoo odoo.addons.base.models.ir_ui_view: A <i> with fa class (fa fa-arrow-right mx-3 text-muted) must have title in its tag, parents, descendants or have text
View error context:
{'file': '/__w/account-invoicing/account-invoicing/account_invoice_payment_term_date_due/views/account_move.xml',
 'line': 3,
 'name': 'account.move form',
 'view': ir.ui.view(1445,),
 'view.model': 'account.move',
 'view.parent': ir.ui.view(668,),
 'xmlid': 'view_move_form'} 
```

While you're at this, could you solve this warning?
it's not hurting anyone, but still..

Reply:
Done! I added the line `title="Arrow icon"`